test/repl.jl

```diff
@@ -348,7 +348,7 @@ begin
     LineEdit.accept_result(s, histp)
     @test LineEdit.mode(s) == repl_mode
     @test buffercontents(LineEdit.buffer(s)) == "shell"
-    @test position(LineEdit.buffer(s)) == 4
+    @test position(LineEdit.buffer(s)) == 2
```

Comment:
TBH, I don't know and I couldn't really figure out what this was even supposed to be testing. @Keno, do you have any idea what's up here?